app.js

```diff
@@ -19,5 +19,14 @@ app.put('/donuts/:id',(req,res)=>{
 });
 
 
+app.post('/donuts',(req,res)=>{
+const donut={
+  id:donuts.length+1,
+  type:req.body.type,
+  cost:req.body.cost
+}
+
+
```

Comment:
This code doesn't seem to be working properly.
where the rest of the function.

Reply:
i updated the post function